errors.go

```diff
@@ -123,5 +123,5 @@ type ErrNumericLiteralFloat struct {
 }
 
 func (e *ErrNumericLiteralFloat) Error() string {
-	return "literal numeric floats are not allowed"
```

Comment:
This is related to the last float PR. it is just an improvement on the error message to show the float value.